### ISSUE TITLE
Cant open Web-UI from certain Networks with some Policy restrictions

### ISSUE THREAD

**Issue by nEINSko** [oznu/homebridge-config-ui-x#517]
Hey there!
First of all, thanks for this Plugin! Makes life so much easier!
I hooked up my pi to a DynDns Service and can reach it from outside of my local network!

After beeing Happy that i can access it from everywhere, i tried opening it at my work.
Unfortunately i could not open the page.

My collegue from the IT department told me there was an error message showing in the logs.

**Logs:**

```
Access to XMLHttpRequest at 'http://xxxxxxxx' (redirected from 'http://xxxxx/api/auth/settings') from origin 'null' has been blocked by CORS policy: The 'Access-Control-Allow-Origin' header contains the invalid value 'false'.
```

**Homebridge Config:**

```json
Show your homebridge config.json here
```
**Environment:**

* **Node.js Version**: v13.6.0
* **NPM Version**: v6.13.6
* **Homebridge Config UI X Version**: 0.4.50
* **Operating System**: Raspbian

**Comment by oznu** [comment]
> CORS policy: The 'Access-Control-Allow-Origin' header contains the invalid value 'false'.

CORS is not required, nor should be required, when using the UI.

I don't know what your work place is doing so not sure how to help here sorry.

**Comment by nEINSko** [comment]
okay, so i guess, i will use it everywhere, except work.

thanks for your answer!

**Comment by oznu** [comment]
If you work out the issue and a change is needed on my end, let me know and I'll see if it's something I can implement.

**Comment by daltonrowe** [comment]
I ran into CORS errors from fresh install, accessed via Chrome Version 117.0.xx

I navigated to `chrome://settings/content/siteDetails?site=http%3A%2F%2Fhomebridge.local` which is the local Chrome settings for the `homebridge.local` domain, and set `Insecure Content` to `Allow`. This solved my problem and the Homebridge UI is now properly loading.

It seems that recent Chrome versions don't allow CSS + JS loaded from http local domains.
